File: TavernSim/src/main/java/org/TavernSim/DialogSet.java
package org.TavernSim;
import java.util.*;

public class DialogSet {
    private String[] dialogLines;
    private Random random;

    public DialogSet(String... lines) {
        dialogLines = lines;
        random = new Random();
    }

    public String getRandomDialog() {
        int randomIndex = random.nextInt(dialogLines.length);
        return dialogLines[randomIndex];
    }

}

File: TavernSimCSharp/UiManagement/DialogSet.cs



public class DialogSet : Displayable {
    public String[] dialogLines;
    private Random random;

    public DialogSet(String[] lines){
        dialogLines = lines;
        random = new Random();
    }

    public Displayable getRandomDialog() {
        int randomIndex = random.Next(dialogLines.Length);
        return new Displayable(dialogLines[randomIndex]);
        //This will not work correctly, issue is in Displayable types i guess?`
        //check that back later
    }

    public new string[] Array(){
        return dialogLines;
    }
}